src/sos/workflow_engines.py

```diff
@@ -182,17 +178,20 @@ def purge_workflows(self,
                         tags=None,
                         verbosity=2):
         try:
-            return self.agent.check_output(
-                "{} purge {} {} {} {} {} -v {}".format(
-                    self.agent.config.get("sos", "sos"),
-                    " ".join(workflows),
-                    "--all" if purge_all else "",
-                    f"--age {age}" if age is not None else "",
-                    f'--status {" ".join(status)}'
-                    if status is not None else "",
-                    f'--tags {" ".join(tags)}' if tags is not None else "",
-                    verbosity,
-                ))
+
+            check_output2 = (
+                f"{self.agent.config.get('sos', 'sos')} purge"
+                f"{' '.join(workflows)}"
+                f"{'--all' if purge_all else ''}"
+                f"--age {age}" if age is not None else ''
+                f"--status {' '.join(status)}"
+                if status is not None else ''
+                f"--tags {' '.join(tags) if tags is not None else ''}"
+                f"{verbosity}"
+            )
```

Comment:
Missing `-v`

Reply:
I'll begin resolving this issue right now.

Reply:
I just checked my GitHub page. -v has now been added to line 188 within src/sos/workflow_engines.py.